dashboard/agent.py

```diff
@@ -159,7 +173,13 @@ async def _check_parent():
             # Http server is not started in the minimal version because
             # it requires additional dependencies that are not
             # included in the minimal ray package.
-            self.http_server = await self._configure_http_server(modules)
+            try:
+                self.http_server = await self._configure_http_server(modules)
+            except Exception:
+                logger.exception(
```

Comment:
Can you add TODO here to remove this in the future? (and do better port resolution)